test/scripts/agent-images/build_onboarding_image.sh

```
@@ -0,0 +1,79 @@
+#                 @redhat-et/flightctl-dev). Set to "" plus RPM_DIR to use local
+#                 Fedora RPMs instead.
+#   RPM_COPR_PACKAGE  COPR agent package name (default: flightctl-agent)
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+ROOT_DIR="$(cd "$SCRIPT_DIR/../../.." && pwd)"
+
+OS_ID="fedora-bootc"
+export AGENT_OS_ID="${OS_ID}"
+export OS_ID
+
+SOURCE_GIT_TAG="${SOURCE_GIT_TAG:-$(${ROOT_DIR}/hack/current-version)}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)